content/guides/models/tables/log_tables.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Log tables
+weight: 1
+---
+
+Visualize and log tabular data with W&B Tables. A table is a two-dimensional grid of data where each column has a single type of data. Each row represents one or more data points logged to a W&B [run]({{< relref "/guides/models/track/runs/" >}}). Tables support primitive and numeric types, as well as nested lists, dictionaries, and rich media types.
```

Comment:
```suggestion
Visualize and log tabular data with W&B Tables. A W&B Table is a two-dimensional grid of data where each column has a single type of data. Each row represents one or more data points logged to a W&B [run]({{< relref "/guides/models/track/runs/" >}}). Tables support primitive and numeric types, as well as nested lists, dictionaries, and rich media types.
```
Trying to disambiguate but I wonder if you should say that W&B Table and "table" are the same thing in this page.

Reply:
Proper vs common noun (The W&B Table product vs a table).